scripts/ibutsu-pod.sh

```diff
@@ -12,13 +82,15 @@ podman run -dt \
        -e POSTGRES_USER=ibutsu \
        -e POSTGRES_DB=ibutsu \
        -e POSTGRES_PASSWORD=ibutsu \
+       $POSTGRES_EXTRA_ARGS \
```

Comment:
At https://hub.docker.com/_/postgres they also recommend storing the data under another folder in `/var/lib/postgres/data`. See the section titled `PGDATA`. We could easily add that with this var.

Reply:
OK, I've specified the PGDATA variable.